### PR TITLE
SignalR bug fix for null pointer exception

### PR DESCRIPTION
**Pull request by sergey-shandar**
## Description

Similar to this PR https://github.com/Azure/azure-powershell/pull/6064

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by sergey-shandar** [comment]
Signed build https://azuresdkci.westus2.cloudapp.azure.com/job/ps-sign/585/

File: src/ResourceManager/SignalR/ChangeLog.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## 0.1.1
+
```
Comment (sergey-shandar) [inline src/ResourceManager/SignalR/ChangeLog.md]: @cormacpayne Is it ok to have a changelog for preview?

**Comment by vladimir-shcherbakov** [comment]
Sing build: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/587/
On demand build: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/814/

File: src/ResourceManager/SignalR/Commands.SignalR/AzureRM.SignalR.psd1 [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.1.1'
 
 # Supported PSEditions
 # CompatiblePSEditions = @()
```
Comment (cormacpayne) [inline src/ResourceManager/SignalR/Commands.SignalR/AzureRM.SignalR.psd1]: @sergey-shandar update the following items in this file:
- Update the version of `AzureRM.Profile` to `5.0.0`
- Update the `ReleaseNotes` at the bottom with the new change log information

**Pull request merged**